packages/docusaurus/src/webpack/utils.ts

```diff
@@ -85,71 +85,30 @@ export function getCacheLoader(
   };
 }
 
-export function getBabelLoader(isServer: boolean, babelOptions?: {}): Loader {
-  const absoluteRuntimePath = path.dirname(
-    require.resolve(`@babel/runtime/package.json`),
-  );
-  return {
-    loader: require.resolve('babel-loader'),
-    options: Object.assign(
+export function getBabelLoader(
+  isServer: boolean,
+  babelOptions?: TransformOptions | string,
```

Comment:
I think it's not worth to keep the babelOptions object here, as it's an internal method and it won't ever be called with an object.

I'd rather have:

```
export function getBabelLoader({isServer, configFile}: {isServer: boolean, configFile?: string}) {

}
```

Reply:
It's not completely internal. It has been exposed here: https://v2.docusaurus.io/docs/lifecycle-apis#configurewebpackconfig-isserver-utils

Reply:
ah :) that's fine then, I guess we need to be retrocompatible